openssl/src/cipher_ctx.rs

```diff
@@ -102,6 +102,15 @@ impl CipherCtx {
             Ok(CipherCtx::from_ptr(ptr))
         }
     }
+
+    #[corresponds(EVP_CIPHER_CTX_copy)]
+    pub fn clone(&self) -> Result<Self, ErrorStack> {
+        let n = CipherCtx::new()?;
+        unsafe {
+            cvt(ffi::EVP_CIPHER_CTX_copy(n.as_ptr(), self.as_ptr()))?;
```

Comment:
I think I'd probably make this a `pub fn copy(&mut self, other: &CipherCtxRef) -> Result<(), ErrorStack>` to follow the OpenSSL API.

Reply:
Done.